maint-lib/stage.py

```diff
@@ -45,10 +45,26 @@
     sys.exit(1)
 
 
+def populateCephPointRelease():
+    # We need to extract a possible CEPH_POINT_RELEASE from CEPH_VERSION
+    # It could be under the two following syntax :
+    # - package_name-package_version
+    # - package_name=package_version"
+    os.environ["CEPH_POINT_RELEASE"] = ""
+    if os.environ["CEPH_VERSION"]:
```

Comment:
Use the preexisting method `getEnvVar` to both get the env var and throw error if it doesn't exist. Suggest looking at and modeling this method after `exportGoArchEnvVar()`

Reply:
Ok. Fixed this way.